tests/longformer/test_modeling_longformer.py

```diff
@@ -416,11 +416,11 @@ def test_diagonalize(self):
 
     def test_pad_and_transpose_last_two_dims(self):
         hidden_states = self._get_hidden_states()
-        self.assertTrue(hidden_states.shape, (1, 8, 4))
+        self.assertEqual(hidden_states.shape, (1, 4, 8))
```

Comment:
This was recently correct by @ydshieh if I remember correctly

Reply:
**[Just to add some context]**

The PR #15537 didn't touch the details like `LongformerSelfAttention._pad_and_transpose_last_two_dims`.

In the simplest words, that PR just performed `unpadding` in the final outputs in `(TF)LongformerEncoder`.

----------------------------
Thanks again, @code-review-doctor !